### PR TITLE
Prevent parent scroll

### PR DESCRIPTION
**Pull request by jmmendivil**
This prevent parent element to scroll when current sticky element ends, it works for fit elements only _(playlist)_

### CODE REVIEW

File: scroll-kit.coffee [hunk 1]
```diff
@@ -330,6 +339,9 @@ initialize_sticky = (node) ->
   node.isFixed = data.fixed or (el.css('position') is 'fixed')
   node.placeholder = placeholder(node) if update_sticky(node)
 
+  if node.data.fit
+    node.el.on('DOMMouseScroll mousewheel', preventScroll)
```
Comment (pateketrueke) [inline scroll-kit.coffee]: I'm not sure if attaching this event per-sticky is a good idea, my suggestion is attaching only a global handler for the whole body, then, identify which sticky is within this handler and prevent, etc.

Comment (jmmendivil) [inline scroll-kit.coffee]: Maybe we should return to this.
Current implementation only works for element first parent-children relationship _(`el.parentElement` once; think in an `e.target` that's not first children like an anchor inside a span, inside a div, inside the sticky element)_, one solution could be [search for sticky parent recursively](https://github.com/jmmendivil/scroll-kit/blob/master/scroll-kit.coffee#L102-L108) but I'm not sure that's a better approach than this.

Comment (pateketrueke) [inline scroll-kit.coffee]: Makes sense, did you considered using `pointer-events` for this?

I remember you can pass or deny those pointer events, such as scroll, using css only...

Comment (jmmendivil) [inline scroll-kit.coffee]: `pointer-events`? so disable events _(even click)_, what do you have in mind?

**Pull request merged**